### PR TITLE
profile for user

### PR DESCRIPTION
**Pull request by at-trinhle**

### CODE REVIEW

File: final-app/src/styles/pages/_profile.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.profile-user {
+  position: relative;
+  margin: 100px 20px;
+  display: flex;
```
Comment (at-dongho) [inline final-app/src/styles/pages/_profile.scss]: - Add prefix when using property flexbox
- Ex: -webkit-

File: final-app/src/styles/pages/_profile.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    width: 30%;
+    padding: 0px 60px;
+    text-align: center;
+    .img-avatar {
+      margin-bottom: 15px;
+      &:hover{
+        cursor: pointer;
+        opacity: 0.9;
+        border-radius: 50%;
+        transition-property: all;
```
Comment (at-dongho) [inline final-app/src/styles/pages/_profile.scss]: - Can write shorthand `transition: all .3s;`

File: final-app/src/styles/pages/_profile.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      padding: 10px 20px;
+    }
+  }
+  .statistical {
+    .total-sumary {
+      padding: 25px 25px;
+      border: 2px solid map-get($color, secondary);
+      margin: 15px 0px;
+      background-color: map-get($color, primary);
+      p{
```
Comment (at-dongho) [inline final-app/src/styles/pages/_profile.scss]: space

File: final-app/src/app/profile/profile.component.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<section class="profile-user">
+  <div class="profile-img">
+    <img class="img-avatar" src="../../assets/images/img_avatar2.png" alt="avata-user">
+    <div class="box-content"></div>
+    <input class="btn" type="button" value="Update image">
+  </div>
+  <div class="main-profile">
```
Comment (at-dongho) [inline final-app/src/app/profile/profile.component.html]: change class to `profile-main`

**Comment by at-trinhle** [comment]
delete profile for user

**Pull request closed without merging**